### PR TITLE
remove cert-manager

### PR DESCRIPTION
**Pull request by rakeshgm**
we do not need cert-manager to be deployed since
webhooks have been removed

### CODE REVIEW

File: test/envs/regional-dr.yaml [hunk 1]
```diff
@@ -50,7 +50,6 @@ templates:
       - addons:
           - name: ocm-hub
           - name: ocm-controller
-          - name: cert-manager
```
Comment (nirs) [inline test/envs/regional-dr.yaml:53]: We have this addon in other environments, need to be removed from all.

Also we want to remove the addon. If there will be a need to use the addon in the future we can get it from earlier version. Maybe keep the addon removal in a separate commit to make it easy to revert if needed.

Comment (rakeshgm) [reply test/envs/regional-dr.yaml:53]: done

**Review by nirs — COMMENTED** [review]
Looks good, I'll just run basic-test with deployment and kubevirt vm to make sure cert-manager is not needed by other component.

**Comment by nirs** [comment]
Tested with basic-test and regional dr:

```
$ basic-test/run -c deployment-k8s-regional-rbd.yaml envs/regional-dr.yaml 
2024-03-20 18:28:40,998 INFO    [deploy] Deploying application
2024-03-20 18:28:40,998 INFO    [deploy] Deploying application 'deployment-rbd'
2024-03-20 18:28:42,247 INFO    [deploy] Waiting for 'placement.cluster.open-cluster-management.io/placement' decisions
2024-03-20 18:28:42,463 INFO    [deploy] Application running on cluster 'dr1'
2024-03-20 18:28:42,598 INFO    [enable-dr] Enable DR
2024-03-20 18:28:42,650 INFO    [enable-dr] Disabling OCM scheduling for 'placement.cluster.open-cluster-management.io/placement'
2024-03-20 18:28:42,796 INFO    [enable-dr] Waiting for 'placement.cluster.open-cluster-management.io/placement' decisions
2024-03-20 18:28:43,246 INFO    [enable-dr] waiting for namespace deployment-rbd
2024-03-20 18:28:43,400 INFO    [enable-dr] Waiting until 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' reports status
2024-03-20 18:28:43,817 INFO    [enable-dr] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' Available condition
2024-03-20 18:28:43,999 INFO    [enable-dr] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' PeerReady condition
2024-03-20 18:28:44,165 INFO    [enable-dr] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' first replication
2024-03-20 18:30:13,111 INFO    [enable-dr] DR enabled
2024-03-20 18:30:13,260 INFO    [failover] Fail over application
2024-03-20 18:30:13,323 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' Available condition
2024-03-20 18:30:13,502 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' PeerReady condition
2024-03-20 18:30:13,680 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' first replication
2024-03-20 18:30:13,818 INFO    [failover] Waiting for 'placement.cluster.open-cluster-management.io/placement' decisions
2024-03-20 18:30:14,116 INFO    [failover] Starting failover for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' to cluster 'dr2'
2024-03-20 18:30:14,286 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' Available condition
2024-03-20 18:30:43,227 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' PeerReady condition
2024-03-20 18:33:13,186 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' first replication
2024-03-20 18:33:13,300 INFO    [failover] Application was failed over
2024-03-20 18:33:13,488 INFO    [relocate] Relocate application
2024-03-20 18:33:13,552 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' Available condition
2024-03-20 18:33:13,735 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' PeerReady condition
2024-03-20 18:33:13,918 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' first replication
2024-03-20 18:33:14,047 INFO    [relocate] Waiting for 'placement.cluster.open-cluster-management.io/placement' decisions
2024-03-20 18:33:14,364 INFO    [relocate] Starting relocate for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' to cluster 'dr1'
2024-03-20 18:33:14,553 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' phase 'Relocated'
2024-03-20 18:35:13,195 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' Available condition
2024-03-20 18:35:13,383 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' PeerReady condition
2024-03-20 18:35:43,191 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc' first replication
2024-03-20 18:35:43,265 INFO    [relocate] Application was relocated
2024-03-20 18:35:43,411 INFO    [disable-dr] Disable DR
2024-03-20 18:35:43,462 INFO    [disable-dr] Deleting 'drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc'
2024-03-20 18:36:13,107 INFO    [disable-dr] Enabling OCM scheduling for 'placement.cluster.open-cluster-management.io/placement'
2024-03-20 18:36:13,181 INFO    [disable-dr] DR was disabled
2024-03-20 18:36:13,292 INFO    [undeploy] Deleting application
2024-03-20 18:36:13,292 INFO    [undeploy] Undeploying application 'deployment-rbd'
2024-03-20 18:36:19,653 INFO    [undeploy] Application was deleted
```

**Comment by nirs** [comment]
Tested also with regional-dr-kubevirt setup:

```
$ basic-test/run -c vm-pvc-k8s-regional.yaml envs/regional-dr.yaml 
2024-03-20 19:01:15,981 INFO    [deploy] Deploying application
2024-03-20 19:01:15,981 INFO    [deploy] Deploying application 'vm-pvc'
2024-03-20 19:01:17,263 INFO    [deploy] Waiting for 'placement.cluster.open-cluster-management.io/placement' decisions
2024-03-20 19:01:17,518 INFO    [deploy] Application running on cluster 'dr1'
2024-03-20 19:01:17,700 INFO    [enable-dr] Enable DR
2024-03-20 19:01:17,774 INFO    [enable-dr] Disabling OCM scheduling for 'placement.cluster.open-cluster-management.io/placement'
2024-03-20 19:01:17,969 INFO    [enable-dr] Waiting for 'placement.cluster.open-cluster-management.io/placement' decisions
2024-03-20 19:01:18,481 INFO    [enable-dr] waiting for namespace vm-pvc
2024-03-20 19:01:18,687 INFO    [enable-dr] Waiting until 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' reports status
2024-03-20 19:01:19,134 INFO    [enable-dr] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' Available condition
2024-03-20 19:01:19,339 INFO    [enable-dr] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' PeerReady condition
2024-03-20 19:01:19,571 INFO    [enable-dr] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' first replication
2024-03-20 19:02:48,215 INFO    [enable-dr] DR enabled
2024-03-20 19:02:48,333 INFO    [failover] Fail over application
2024-03-20 19:02:48,382 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' Available condition
2024-03-20 19:02:48,543 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' PeerReady condition
2024-03-20 19:02:48,710 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' first replication
2024-03-20 19:02:48,831 INFO    [failover] Waiting for 'placement.cluster.open-cluster-management.io/placement' decisions
2024-03-20 19:02:49,076 INFO    [failover] Starting failover for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' to cluster 'dr2'
2024-03-20 19:02:49,208 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' Available condition
2024-03-20 19:03:18,189 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' PeerReady condition
2024-03-20 19:05:48,176 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' first replication
2024-03-20 19:06:18,181 INFO    [failover] Application was failed over
2024-03-20 19:06:18,298 INFO    [relocate] Relocate application
2024-03-20 19:06:18,342 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' Available condition
2024-03-20 19:06:18,500 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' PeerReady condition
2024-03-20 19:06:18,668 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' first replication
2024-03-20 19:06:18,777 INFO    [relocate] Waiting for 'placement.cluster.open-cluster-management.io/placement' decisions
2024-03-20 19:06:19,036 INFO    [relocate] Starting relocate for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' to cluster 'dr1'
2024-03-20 19:06:19,158 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' phase 'Relocated'
2024-03-20 19:08:18,434 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' Available condition
2024-03-20 19:08:18,619 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' PeerReady condition
2024-03-20 19:08:48,273 INFO    [relocate] Waiting for 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc' first replication
2024-03-20 19:08:48,385 INFO    [relocate] Application was relocated
2024-03-20 19:08:48,573 INFO    [disable-dr] Disable DR
2024-03-20 19:08:48,660 INFO    [disable-dr] Deleting 'drplacementcontrol.ramendr.openshift.io/vm-pvc-drpc'
2024-03-20 19:09:18,256 INFO    [disable-dr] Enabling OCM scheduling for 'placement.cluster.open-cluster-management.io/placement'
2024-03-20 19:09:18,362 INFO    [disable-dr] DR was disabled
2024-03-20 19:09:18,581 INFO    [undeploy] Deleting application
2024-03-20 19:09:18,581 INFO    [undeploy] Undeploying application 'vm-pvc'
2024-03-20 19:09:24,776 INFO    [undeploy] Application was deleted
```

**Review by nirs — APPROVED** [review]

**Review by ShyamsundarR — APPROVED** [review]

**Pull request merged**